src/TraceEvent/Parsers/InteropTraceEventParser.cs

```diff
@@ -0,0 +1,1263 @@
+﻿using System;
```

Comment:
Don't put this file in TraceEvent (it is not generally useful enough).   Put in in PerfView directory (you will see a PerfViewTRaceEventParser.cs there as well).